src/index.html

```diff
@@ -6,14 +6,561 @@
+            </div>
+          </div>
+          <div class="shop__tab_button button">
+            <a
+              href="./shop.html"
+              class="shop__tab_link button--link"
+            >
+              all products
+            </a>
+          </div>
+        </div>
+      </section>
+
+      <section
+        class="about-creator"
+        id="about"
+      >
+        <div class="about-creator_description">
+          <h2 class="title about-creator_description-title">
+            about the
+            <span>creators</span>
+          </h2>
+          <p class="about-creator_description-text">
+            My name is Victoria, I’m a master at candle making and natural
+            cosmetics. In the past, a cynologist, was engaged in herbs, made
+            layouts on the Tarot.
+          </p>
+          <p class="about-creator_description-text">
+            I am also a volunteer to save wild birds in distress, a consultant
+            to the Phoenix Wildlife Rehabilitation Center, a member of the
+            Society for the Conservation of Birds of Ukraine with the support of
+            Birdlife International.
+          </p>
+          <p class="about-creator_description-text--bigger-screen">
+            My husband and I live in the Carpathian region in the city of
+            Chernivtsi. I dream of my center for the rehabilitation of wild
+            birds, as it is very difficult to rescue birds in a rented communal
+            apartment.
+          </p>
+
+          <a
+            href="#"
+            class="about-link"
+          >
+            LEARN MORE
+          </a>
+          <div class="about-creator_button button--bigger-screen button">
+            <a
+              href="#contact"
+              class="about-creator_button-link button--link"
+            >
+              to contact us
+            </a>
+          </div>
+        </div>
+        <div class="about-creator_image">
+          <img
+            src="images/img-creator-tablet.png"
+            alt="bird image"
+            class="about-creator_image-photo--tablet photo_hover"
+          />
+          <img
+            src="images/img-creator-desktop.png"
+            alt="bird image"
+            class="about-creator_image-photo--desktop photo_hover"
+          />
+        </div>
+      </section>
+
+      <section class="powder">
+        <div class="powder_photos">
+          <h2 class="title">
+            Ayurvedic Tooth
+            <br />
+            Powder
+          </h2>
+          <img
+            src="./images/tooth-powder-mobile.png"
+            alt="imagem tooth powder"
+            class="powder_photos-mobile"
+          />
+          <img
+            src="./images/tooth-powder-tablet.svg"
+            alt="imagem tooth powder"
+            class="powder_photos-tablet"
+          />
+        </div>
+        <a
+          href="#"
+          class="about-link about-link--bigger-screen"
+        >
+          LEARN MORE
```

Comment:
i suggest that you change the capitalization of all texts to css and not html, since is not a very good practice.